.gitignore

```diff
@@ -43,3 +43,9 @@ tests/venv
 .cache
 
 docs/generated/
+
+/core/cache-main
```

Comment:
@ewencp I think these are likely coming from the scala plugin, that's why they do not appear under the other projects